### PR TITLE
FEATURE - Add price ticker to broker

### PR DESCRIPTION
**Pull request by dannypaz**
## Description
This PR adds an endpoint and CLI command (`sparkswap info market-stats`) that provides the previous data based off of the past 24 hours from the orderbook. Also referred to as "price ticker". 

Additionally this call can be used w/ integrations such as CCXT

#### Price Ticker Information

All values are in quote/counter currency IE LTC

1. datetime - current datetime stamp (possibly in nano)
2. high - highest price for the past 24 hours 
3. low - lowest price for the past 24 hours
4. bid - current best bid
5. ask - current best ask
6. vwap - add up price of all trades (fills) and divide by amount of trades made in the same time period
7. baseVolume - base amount traded in last 24 hours
8. quoteVolume - counter amount traded in last 24 hours

Changes Include:
1. added `nano-seconds` to the broker repo
2. adding a new cli command `sparkswap info market-stats`
3. adding a new grpc service under the `InfoService`
4. adding a `MarketStats` class used for data calculations

## Related PRs
List related PRs if applicable

## Todos
- [ ] Tests
- [x] Documentation
- [x] Link to Trello

### CODE REVIEW

**Comment by dannypaz** [comment]
[market summary info (fetchTicker) [ccxt]](https://trello.com/c/9BNWQ8n1/659-market-summary-info-fetchticker-ccxt)

File: broker-daemon/broker-rpc/info-service/get-market-stats.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  const currentBids = currentOrderbookEvents.filter(e => e.side === BlockOrder.SIDES.BID)
+  const bestBidPrice = await stats.highestPrice(currentBids)
+  const bestBidAmount = await stats.bestBidAmount(currentBids)
+
+  console.log(currentAsks)
+
+  // We only care about events that have been filled, because highest/lowest prices
+  // and totals are only applicable for orders that are completed
+  // TODO: Do we care about other events IE w/ fill amount?
+  // const filledMarketEvents = currentMarketEvents.filter(e => e.eventType === MarketEvent.TYPES.FILLED)
```
Comment (martineehrlich) [inline broker-daemon/broker-rpc/info-service/get-market-stats.js]: Why is this commented out? Don't we want to filter by filled?

File: broker-daemon/orderbook/index.js [hunk 2]
```diff
@@ -150,6 +150,36 @@ class Orderbook {
...
+   *
+   * @param {String} timestamp - timestamp in nano-seconds
+   * @returns {Array<MarketEventOrder>}
+   */
+  async getOrderbookEventsByTimestamp (timestamp) {
+    return getRecords(
+      this.store,
+      (key, value) => JSON.parse(value),
+      // Limits the query to gte to a specific timestamp
+      MarketEvent.rangeFromTimestamp(timestamp)
```
Comment (martineehrlich) [inline broker-daemon/orderbook/index.js]: When do we use a MarketEvent vs a MarketEventOrder?

Comment (dannypaz) [reply broker-daemon/orderbook/index.js]: MarketEvent is data that comes from the relayer. When we add the information to the orderbook, and then subsequently search for it, we serialize that data to MarketEventOrder.

```
MarketEvent = events from relayer
MarketEventOrder = anything on the orderbook (sent to the client)
```

ill add this information to each class. Additionally, the store that is being used in this case, is `eventStore` which is `MarketEvents`, where as the orderbookStore is `MarketEventOrders`

**Pull request merged**